internal/app/machined/pkg/controllers/runtime/pcirebind_config_test.go

```diff
@@ -0,0 +1,64 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package runtime_test
+
+import (
+	"testing"
+
+	"github.com/cosi-project/runtime/pkg/resource/rtestutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/siderolabs/talos/internal/app/machined/pkg/controllers/ctest"
+	runtimectrls "github.com/siderolabs/talos/internal/app/machined/pkg/controllers/runtime"
+	"github.com/siderolabs/talos/pkg/machinery/config/container"
+	"github.com/siderolabs/talos/pkg/machinery/config/types/pci"
+	"github.com/siderolabs/talos/pkg/machinery/resources/config"
+	"github.com/siderolabs/talos/pkg/machinery/resources/runtime"
+)
+
+type PCIRebindConfigSuite struct {
+	ctest.DefaultSuite
+}
+
+func TestPCIRebindConfigSuite(t *testing.T) {
+	suite.Run(t, new(PCIRebindConfigSuite))
+}
+
+func (suite *PCIRebindConfigSuite) TestPCIRebindConfig() {
+	suite.Require().NoError(suite.Runtime().RegisterController(&runtimectrls.PCIRebindConfigController{}))
+
+	pciRebindConfig := &pci.RebindConfigV1Alpha1{
+		MetaName:          "ixgbe-bind",
+		PCIVendorDeviceID: "0000:04:00.00",
```

Comment:
Ah, well, get it now. Then I believe it should be renamed to path or sth like that in other places as well, since this is not specific to device model but its location within the system (controller/port/switches if any)

Reply:
I think this is usually called just PCI ID? 

it has nothing to do with vendor, but basically a path to the device in the current PCI topology